lambdas/data-migration2/src/granulesAndFiles.ts

```diff
@@ -266,14 +292,21 @@ export const migrateGranulesAndFiles = async (
 
   /* eslint-disable no-await-in-loop */
   while (record) {
-    // eslint-disable-next-line max-len
-    const result = await migrateGranuleAndFilesViaTransaction(record, migrationResult, knex, loggingInterval);
+    const result = await migrateGranuleAndFilesViaTransaction({
+      dynamoRecord: record,
+      granuleAndFileMigrationResult: migrationResult,
+      knex,
+      loggingInterval,
+      stream: jsonWriteStream,
+    });
     migrationResult.granulesResult = result.granulesResult;
     migrationResult.filesResult = result.filesResult;
 
     await searchQueue.shift();
     record = await searchQueue.peek();
   }
+  jsonWriteStream.end();
```

Comment:
It seems like we might need to bring back the close stream helper. I'm getting intermittent unit test failures locally without it which I think are related. Sorry for asking you to remove that.

We just don't need to pass any arguments to the `stream.end()` call.